src/clarity-angular/dropdown/dropdown.ts

```diff
@@ -15,17 +15,15 @@ import {
 import {Point} from "../popover/popover";
 import {menuPositions} from "./menu-positions";
 
-// TODO: the ng-content inside the dropdown-menu should ideally just be
+// TODO: the ng-content inside the ng-template should ideally just be
 // <ng-content select="clr-dropdown-menu"></ng-content>. Remove .dropdown-menu in 1.0?
 @Component({
     selector: "clr-dropdown",
     template: `
         <ng-content select="[clrDropdownToggle]"></ng-content>
         <ng-template [(clrPopover)]="open" [clrPopoverAnchor]="anchor" [clrPopoverAnchorPoint]="anchorPoint"
                      [clrPopoverPopoverPoint]="popoverPoint">
-            <div class="dropdown-menu">
-                <ng-content select="[clr-dropdown-menu, .dropdown-menu]"></ng-content>
-            </div>
+            <ng-content select="[clr-dropdown-menu, .dropdown-menu]"></ng-content>
```

Comment:
again a question. I think we have both selectors here for backward compatibility but eventually we will get rid of the css selector. is that correct?